packages/web/package.json

```diff
@@ -27,6 +27,7 @@
     "classnames": "^2.3.1",
     "dayjs": "^1.10.7",
     "eventemitter3": "^4.0.7",
+    "mobx": "^6.3.10",
```

Comment:
Where is this referenced? If possible we should stick with `mobx-react-lite` to keep bundles small (even though it's used in stores)

Reply:
I guess `mobx-react-lite` is just for React functional component. It can't be used for class definition. And `mobx` is now used in `web/stores/chain.ts`. It is likely to continue to be used in stores that will only be used on `web` package.